test.js

```diff
@@ -9,12 +10,22 @@ test('tempWrite(string)', async t => {
 	t.is(path.basename(filepath), 'test.png');
 });
 
+test('tempWrite(stream)', async t => {
+	const readable = new stream.Readable({
+		read() { /* noop */ }
+	});
```

Comment:
Tests passed without, but I agree it's safer to follow the docs and leave it in.